src/RoomsSDKAdapter.js

```diff
@@ -134,4 +144,119 @@ export default class RoomsSDKAdapter extends RoomsAdapter {
 
     return this.getRoomObservables[ID];
   }
+
+  /**
+   * Invoke request to listActivities on adapter
+   *
+   * @param {object} Params to send to conversation api method
+   * @returns
+   * @private
+   */
+  fetchActivities({roomId, earliestActivityDate}) {
+    const {limit} = this;
+    const conversationId = deconstructHydraId(roomId).id;
```

Comment:
Only works after you use `internal.conversation.list()` as the following methods did not succeed as well:

```
const roomId =
  'Y2lzY29zcGFyazovL3VzL1JPT00vYmMyMjY2YjAtZDZjMy0xMWViLWFlZjUtNmQ3NzkwOGJmY2Ji';
const hydraRoomId = deconstructHydraId(roomId);

console.log(webex.internal.conversation.getConvoUrl(hydraRoomId));
console.log(webex.internal.conversation.getUrlFromClusterId(hydraRoomId));

```

Reply:
You should use the cluster information as well, it will hopefully won't require calling `list()` first.

Reply:
I have tried every possible way to not have to invoke `list()` to pre-populate the cache before request. I have even stepped thru the code and it seems that something is incorrect. Can you provide an example of how to actually invoke the method in which you are stating.